pybossa/api/__init__.py

```diff
@@ -564,6 +564,27 @@ def cancel_task(task_id=None):
                 .format(project.id, current_user.id, task_id))
             release_reserve_task_lock_by_id(project.id, task_id, current_user.id, timeout, expiry=EXPIRE_LOCK_DELAY)
 
+    return Response(json.dumps({'success': True}), 200, mimetype="application/json")
+
+
+@jsonpify
+@csrf.exempt
+@login_required
+@blueprint.route('/task/<int:task_id>/release_category_locks', methods=['POST'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def release_category_locks(task_id=None):
```

Comment:
Since `/<int:task_id>/` is specified in the route, can `task_id` be None at the time? Is there any use case we pass `None` to `task_id`?

Reply:
We cannot pass None. We still need the task_id as it's used to ```release_reserve_task_lock_by_id``` to get the "reserve_key"